bin/diff_yaml.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python
```

Comment:
does this need the copyright header?  In projects past, the legal peeps indicated the LICENSE file applies to all works in a project, however, someone may have a desire to pull this tool out of the repo and reuse it for some other purpose which might be difficult without a license header.

Reply:
if you don't think a copyright header is necessary please let me know, and I'll approve the PR.

Cheers
-steve